src/components/pages/Login/LoginContainer.js

```diff
@@ -7,9 +7,8 @@ import '../../../styles/login.less';
 import { config } from '../../../utils/oktaConfig';
 
 const LoginContainer = () => {
-  let history = useHistory();
-
-  useEffect(() => {
+  const history = useHistory();
+  const loadWidget = useCallback(() => {
```

Comment:
Nice job fixing the linter warning by moving refactoring as a useCallback! Great solution.